src/com/epam/Main.java

```diff
@@ -0,0 +1,8 @@
+package com.epam.training;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Chat.startChat(1);
```

Comment:
remove arrgument from startChat, as it should be given from command line (memCounts)